### PR TITLE
welcome team members

### PR DESCRIPTION
**Pull request by julesgsmith**
inviting rest of team to see the code and review the paragraph

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,6 +13,7 @@ <h1>Hello There Syntax Squad!</h1>
         Feel free to make any changes to this file and create a pull request.
       </p>
```
Comment (lazycloud0) [inline index.html:14]: want to delete this?

**Review by lazycloud0 — CHANGES_REQUESTED** [review]
I've seen it

File: index.html (hunk 1 above)
Comment (julesgsmith) [reply index.html:14]: This is done

**Review by lazycloud0 — APPROVED** [review]
It's all good!

**Pull request merged**